backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/ReleaseService.java

```diff
@@ -19,22 +31,149 @@ public class ReleaseService implements ReleaseAnnouncementUseCase {
 
   private final LoadAnnouncementByNormEliUseCase loadAnnouncementByNormEliUseCase;
   private final UpdateAnnouncementPort updateAnnouncementPort;
+  private final UpdateOrSaveNormPort updateOrSaveNormPort;
+  private final NormService normService;
+  private final TimeMachineService timeMachineService;
+  private final CreateNewVersionOfNormService createNewVersionOfNormService;
+  private final DeleteQueuedNormsPort deleteQueuedNormsPort;
+  private final DeleteUnpublishedNormPort deleteUnpublishedNormPort;
 
   public ReleaseService(
     LoadAnnouncementByNormEliUseCase loadAnnouncementByNormEliUseCase,
-    UpdateAnnouncementPort updateAnnouncementPort
+    UpdateAnnouncementPort updateAnnouncementPort,
+    UpdateOrSaveNormPort updateOrSaveNormPort,
+    NormService normService,
+    TimeMachineService timeMachineService,
+    CreateNewVersionOfNormService createNewVersionOfNormService,
+    DeleteQueuedNormsPort deleteQueuedNormsPort,
+    DeleteUnpublishedNormPort deleteUnpublishedNormPort
   ) {
     this.loadAnnouncementByNormEliUseCase = loadAnnouncementByNormEliUseCase;
     this.updateAnnouncementPort = updateAnnouncementPort;
+    this.updateOrSaveNormPort = updateOrSaveNormPort;
+    this.normService = normService;
+    this.timeMachineService = timeMachineService;
+    this.createNewVersionOfNormService = createNewVersionOfNormService;
+    this.deleteQueuedNormsPort = deleteQueuedNormsPort;
+    this.deleteUnpublishedNormPort = deleteUnpublishedNormPort;
   }
 
+  /**
+   * Queue all norms that are affected by the given announcement for publishing.
+   * This includes generating a new manifestation of the amending norm, and new expressions for each time boundary of
+   * every target norm. It also creates a new manifestation for the current expression of the target norm so it contains
```

Comment:
Why new manifestation of the target norm (ZF0) if we did create this one already when uploading the amending law and we haven't published it yet?

Reply:
Primarily to set the pointInTimeManifestation date to the current date.